client/src/components/jobPosting/JobPostingStage.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Chip } from '@material-ui/core'
```

Comment:
Andrea found [information about how to minimize bundle size](https://material-ui.com/guides/minimizing-bundle-size/)